crates/solver/src/main.rs

```diff
@@ -208,6 +208,24 @@ async fn main() {
         fee_objective_scaling_factor: args.fee_objective_scaling_factor,
     });
 
+    let market_makable_token_list_configuration = TokenListConfiguration {
+        url: args.market_makable_token_list,
+        update_interval: args.market_makable_token_list_update_interval,
+        chain_id,
+        client: http_factory.create(),
+    };
+    let market_makable_token_list = match TokenList::from_configuration(
+        market_makable_token_list_configuration.clone(),
+    )
+    .await
+    {
+        Ok(token_list) => Some(Arc::new(token_list)),
+        Err(err) => {
+            tracing::error!("Couldn't fetch market makable token list: {}", err);
```

Comment:
Seems weird to me that if the initial request fails for some reason we don't have a list for the rest of the week (and seem to fail silently). I think we should either panic startup in this case or optimistically keep refetching this list on every update cycle.

Reply:
Refetching done in background task.